core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -88,6 +88,8 @@ var ExplorationEditorPage = function() {
   var saveDiscardToggleButton = element(
     by.css('.protractor-test-save-discard-toggle'));
   var saveDraftButton = element(by.css('.protractor-test-save-draft-button'));
+  var TranslationTabFeedbackButton = element(
```

Comment:
Move this to `ExplorationEditorTranslationTab.js`

Reply:
done